docs/testing-components/index.md

```diff
@@ -4,4 +4,5 @@ title: Testing Your Components
 
 *   [Unit Testing](./unit-testing/index.md) - How to start Unit Testing
 *   [Test Driven Development(TDD)](./test-driven-development/index.md) - This document describes the test-first methodology
-we aspire to on the Enact team.
\ No newline at end of file
+*   [Debug Accessibility in Enact](./debug-aria/index.md) - How to debug accessibility in Enact app and sampler
```

Comment:
This is not the testing feature. Let's move the new page to the bottom of the `Accessibility Support in Enact` page.